### DISCUSSION TITLE
Admin interface cannot delete user

### DISCUSSION THREAD

**Discussion by Kruzerson** [dani-garcia/vaultwarden#4546] [Q&A] [answered]
### Subject of the issue
Admin interface cannot delete user.

I upgraded to version 1.30.5 and still can't delete a user through the admin panel. Decatification and other actions are not available either. A similar problem was solved here #4399

### Your environment (Generated via diagnostics page)
* Vaultwarden version: v1.30.5
* Web-vault version: v2024.1.2b
* OS/Arch: linux/x86_64
* Running within a container: true (Base: Debian)
* Environment settings overridden: true
* Uses a reverse proxy: true
* IP Header check: true (X-Real-IP)
* Internet access: true
* Internet access via a proxy: false
* DNS Check: true
* Browser/Server Time Check: true
* Server/NTP Time Check: true
* Domain Configuration Check: true
* HTTPS Check: true
* Database type: MySQL
* Database version: 10.3.39-MariaDB-0ubuntu0.20.04.2
* Clients used: 
* Reverse proxy and version: 
* Other relevant information: 

**Steps to reproduce**
Upgrade to 1.30.5, login to admin, then open browser console. You'll notice the UI isn't responsive in parts where jQuery is used.

**Comment by BlackDex** [comment]
They're are no known issues. It works fine for me using that version. 

You probably have old files in the cache or your reverse proxy is caching soccer stuff.

What do there browser developer console and Vaultwarden logs report?

**Reply by Kruzerson** [reply]
![image](https://github.com/dani-garcia/vaultwarden/assets/130353475/dd2886b3-47d7-4d26-a8c3-6ef493a0eacf)

**Reply by BlackDex** [reply]
That shows that it still tries to download an old none existing version. Probably browser cache. Try a different browser or private mode and see what happens. Also try to restart your reverse proxy.

**Reply by Kruzerson** [reply]
I tried other browsers and an anonymous tab. I also rebuilt the proxy container. it didn't help

**Reply by Kruzerson** [reply]
I also turned off the proxy reverse and it didn't help either

**Reply by BlackDex** [reply, marked as answer]
Did you replaced/overridden the template files? Those also contain admin templates and could cause the issue you have now also.

**Reply by Kruzerson** [reply]
I changed the language of emails using this [instruction](https://github.com/dani-garcia/vaultwarden/wiki/Translating-the-email-templates). 
Thats all.

**Reply by Kruzerson** [reply]
I disabled the templates and you were right, the deletion worked. Can you explain how to work with them when updating?

**Reply by Kruzerson** [reply]
@BlackDex Thank you very much. You have been very helpful. Thank you for your quick response!

**Reply by BlackDex** [reply]
You only have to include the templates which you want to have adjusted. So in theory you can just delete the admin directory and leave the mail one.
